.codeclimate.yml

```diff
@@ -29,11 +33,6 @@ plugins:
     config: ".rubocop_cc.yml"
     channel: rubocop-0-82
 exclude_patterns:
-- config/dictionary_strings.rb
-- config/model_attributes.rb
-- config/yaml_strings.rb
-- db/schema.rb
-- lib/generators/**/templates/
```

Comment:
These were removed by the script, but we may still want this line.  The rest of these files aren't checked in.

Reply:
https://github.com/ManageIQ/manageiq/pull/20916